ppr-ui/src/store/getters/state-getters.ts

```diff
@@ -43,6 +45,10 @@ import { toRefs } from '@vue/composition-api'
 import { MhrSectVal } from '@/composables/mhrRegistration/enums'
 import { getFeatureFlag } from '@/utils'
 
+export const getAccountModel = (state: AccountIF): AccountModelIF => {
+  return state.account
+}
+
```

Comment:
I feel strongly this shouldn't be separate from our stateModel unless there is some major reason why. 

Either way when adding new properties to our stateModel or creating a new model, you need to initialize the state properties in the store in order for inner working of Vue to create those reactivity hooks.

Reply:
This model already existed in the state store, along with other models (auth, notification). I just accessed it because I needed to get `id` under `account.currentAccount`. So everything is already there and initiated.
<img width="289" alt="Screen Shot 2023-01-25 at 3 30 03 PM" src="https://user-images.githubusercontent.com/2333290/214716601-bd66df7d-16f4-4b7d-bccf-f6daa5640895.png">

Reply:
O man, i wonder why. 
Did it have any other accessors? 

There is account information in our state as well 😕